osquery/logger/CMakeLists.txt

```diff
@@ -61,3 +61,12 @@ if(NOT SKIP_AWS AND NOT WINDOWS)
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-sts")
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-core")
 endif()
+
+if(NOT WINDOWS)
```

Comment:
I'm not sure if FreeBSD will like this either. Do you mind using the `SKIP_AWS`-style make logic with this too? Essentially define a `SKIP_KAFKA` CMake variable if the environment variable is set. Then only including these links + implementation code if the variable is not defined?